### PR TITLE
Assigned tasks for Swartz27 in task_list.txt

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by nitrocaster** [comment]
Some minor suggestions:
1] Add some brief description to task #24
2] Regarding pre-compiled shaders issue - it should not be completely disabled because in that case shader compilation takes noticeable amount of time and will occur every time user loads the game

**Comment by ghost** [comment]
1) Will do
2) This only occurs for the very first time a shader has been changed/added. Once the shaders_cache is built when the game starts it never needs to be built again and will always load the game instantly. I've been using this for almost a year so I know this to be the case, it does not affect load times unless a new or changed shader has since been added.

**Comment by nitrocaster** [comment]
Sure. I mean, if you disable shader cache, the game will compile shaders every time, and we don't want this to happen.

**Comment by ghost** [comment]
I don't think I'm explaining this very well, let me try again (as I've been using this system for a long time now):

Step 1: A new or modified shader is added to the project
Step 2: The user deletes their shaders_cache folder
Step 3: The initial time, and the initial time only, the shaders_cache is re-built
Step 4: On all subsequent playthroughs the shaders_cache does not get re-built. It utilizes the shaders_cache that was built that initial time from now on. It does not compile shaders every time, it only compiles shaders that one single time.

I hope that explains it better.

Without doing this new or modified shaders cannot be added to the game. I am not the only modder that has done this.

**Comment by nitrocaster** [comment]
Exactly. That's how it should work, except for one thing: step 2 must be detected by the engine automatically so user wouldn't care about deleting shader cache each time some shader changes.

**Comment by ghost** [comment]
That I can completely agree with you on. I'm not quite sure how to go about that but I'll see if I can figure it out :)
